R/pkg/inst/tests/testthat/test_mllib.R

```diff
@@ -971,10 +971,15 @@ test_that("spark.randomForest Classification", {
   predictions <- collect(predict(model, data))$prediction
   expect_equal(length(grep("1.0", predictions)), 50)
   expect_equal(length(grep("2.0", predictions)), 50)
+
+  # spark.randomForest classification can work on libsvm data
+  data <- read.df(absoluteSparkPath("data/mllib/sample_multiclass_classification_data.txt"),
+                source = "libsvm")
+  model <- spark.randomForest(data, label ~ features, "classification")
+  expect_equal(summary(model)$numFeatures, 4)
```

Comment:
The reason behind this is there is no suitable libsvm data under ```data/mllib/``` for ```naiveBayes``` and ```glm```. Currently SparkR only support ```Bernoulli``` naive bayes which require all features data should be 0 or 1. And the binary classification data under ```data/mllib``` is collinear data, which is not very suitable to illustrate ```glm``` with binomial family. I'm working on another PR to fix this collinear issue, so I added test for ```randomForest``` and ```gbt```.